src/components/ProductPage/NavbarFilter/NavbarFilter.jsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import "./NavbarFilter.scss";
+
+const NavbarFilter = () => {
+	const brandNames = [
+		{
+			name: "Domyos",
+		},
+		{
+			name: "Artengo",
+		},
+		{
+			name: "Quechua",
+		},
+		{
+			name: "Kalenji",
+		},
+	];
+	const categoryNames = [
+		{
+			name: "shirts",
+		},
+		{
+			name: "shoes",
+		},
+		{
+			name: "trousers",
+		},
+		{
+			name: "backpack",
+		},
+		{
+			name: "sunglasses",
+		},
+	];
+	return (
+		<div className='navbarfilter-container'>
+			<div className='spacer-3rem'></div>
+			<div className='category-labels'>
+				<label className='category-heading'>FILTERS</label>
+				<label className='category-heading1'>CLEAR ALL</label>
+			</div>
+			<div className='navbarfilter-price'>
+				<h3 className='navbarfilter-sort'>SORT</h3>
+				<div className='navbarfilter-display'>
+					<label className='navbarfilter-label'>
+						<input className='navbarfilter-checkbox' type='radio' />
+						Price High to Low
+					</label>
+				</div>
+				<div className='navbarfilter-display'>
```

Comment:
Suggestion: Can have the radio button as a component, since it is repeating twice as of now.

Reply:
Okay, I'm converting data into array. I will be inside map function that one only using one radio button changes in the next PR review. thanks.